driver/csiplugin/controllerserver.go

```diff
@@ -2368,6 +2482,11 @@ func (cs *ScaleControllerServer) CreateSnapshot(ctx context.Context, req *csi.Cr
 		return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("CreateSnapshot - Error in source Volume ID %v: %v", volID, err))
 	}
 
+	// Block snapshot for cache volume
+	if volumeIDMembers.StorageClassType == STORAGECLASS_CACHE {
+		return nil, status.Error(codes.InvalidArgument, "taking snapshot of cache volume is not supported")
```

Comment:
shall we change change it to something like
CreateSnapshot - Snapshot of cache volume is not supported

Reply:
Done